source/accessibility-statement.html.erb

```diff
@@ -126,7 +126,7 @@ title: Accessibility statement for GOV.UK&nbsp;Pay
       <p class="govuk-body">We previously tested this service for compliance with WCAG 2.1 AA on 5 September 2019.</p>
 
       <h2 class="govuk-heading-m">What we’re doing to improve accessibility</h2>
-      <p class="govuk-body">Our <a class="govuk-link "href="https://www.payments.service.gov.uk/roadmap/">roadmap</a> includes information about how and when we plan to improve accessibility on this service.</p>
+      <p class="govuk-body">Our <a class="govuk-link "href="/roadmap/">roadmap</a> includes information about how and when we plan to improve accessibility on this service.</p>
```

Comment:
This needs to be: `<a class="govuk-link" href="/roadmap/">`

Reply:
Good spot! Updated https://github.com/alphagov/pay-product-page/compare/11d8eeb92f724e60dfcf0e7bc13ef8d5e91567ea..2abba9afdf0fbeaefdc62d137af2d6697eb6706f